src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List<Bulldozer> get() {
+        Bulldozer model1 = new Bulldozer();
```

Comment:
```suggestion
        Bulldozer modelOne = new Bulldozer();
```
try to avoid digits in names